apps/meteor/client/views/admin/sidebarItems.ts

```diff
@@ -94,28 +94,28 @@ export const {
 			]),
 	},
 	{
-		icon: 'discover',
-		href: 'admin-oauth-apps',
-		i18nLabel: 'OAuth Apps',
-		permissionGranted: (): boolean => hasAllPermission('manage-oauth-apps'),
+		href: 'admin-import',
+		i18nLabel: 'Import',
+		icon: 'import',
+		permissionGranted: (): boolean => hasPermission('run-import'),
 	},
 	{
-		icon: 'mail',
-		href: 'admin-mailer',
-		i18nLabel: 'Mailer',
-		permissionGranted: (): boolean => hasAllPermission('access-mailer'),
+		href: 'admin-view-logs',
+		i18nLabel: 'Logs',
+		icon: 'post',
+		permissionGranted: (): boolean => hasPermission('view-logs'),
 	},
 	{
-		icon: 'user',
-		href: 'user-status',
-		i18nLabel: 'User_Status',
-		permissionGranted: (): boolean => hasAtLeastOnePermission(['manage-user-status']),
+		href: 'custom-sounds',
+		i18nLabel: 'Sounds',
+		icon: 'volume',
+		permissionGranted: (): boolean => hasPermission('manage-sounds'),
 	},
 	{
-		icon: 'lock',
-		href: 'admin-permissions',
-		i18nLabel: 'Permissions',
-		permissionGranted: (): boolean => hasAtLeastOnePermission(['access-permissions', 'access-setting-permissions']),
+		icon: 'emoji',
+		href: 'emoji-custom',
+		i18nLabel: 'Emoji',
+		permissionGranted: (): boolean => hasPermission('manage-emoji'),
 	},
```

Comment:
Not a problem, but in terms of maintaining the the structure of the file, follow the same order for all props... haha